libs/src/service-selection/constants.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  DECISION_TREE_STATE: Object.freeze({
+    CHECK_SHORT_CIRCUIT: 'Check Short Circuit',
+    GET_ALL_SERVICES: 'Get All Services',
+    FILTER_TO_WHITELIST: 'Filter To Whitelist',
+    FILTER_FROM_BLACKLIST: 'Filter From Blacklist',
+    FILTER_OUT_KNOWN_UNHEALTHY: 'Filter Out Known Unhealthy',
+    GET_SELECTION_ROUND: 'Get Selection Round',
+    NO_SERVICES_LEFT_TO_TRY: 'No Services Left To Try',
```

Comment:
when are you dropping
Ain't got no services left to try

Reply:
:O why would we drop dis

Reply:
@ariana guande

Reply:
@arianaguande